addons/mail_client_extension/models/crm_lead.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, models
+
+class Lead(models.Model):
+    _inherit = 'crm.lead'
+
+    @api.model
+    def form_view_auto_fill(self):
```

Comment:
This doesn't need to be public or api.model if it's only used from a server action. Also didn't al want this method to be inlined? I don't remember what the conclusion was.

Reply:
At the end of the day, he didn't really care and pretty much said it was up to my tastes whether to put it inline or not

Reply:
works for me. Still shouldn't need to be callable over RPC though, I think?